src/utils/__tests__/system.test.ts

```diff
@@ -159,29 +165,27 @@ describe('calculateChecksum', () => {
 });
 
 describe('isExecutableInPath', () => {
-  test('checks for existing executable', async () => {
-    expect(hasExecutable('ls')).toBe(true);
+  test('checks for existing executable', () => {
+    expect(hasExecutable('git')).toBe(true);
```

Comment:
It's probably safe to assume, but should we maybe even go safer and assume `node` here?